### PR TITLE
Initial checkin of fortune

### PR DESCRIPTION
**Pull request by berrystephenw**
adding the files for fortune. Let me know if there is a problem!

### CODE REVIEW

File: _plugins/fortune.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+
+## Configuration
+
+The only configuration is to enable or disable the icon on the navigation bar, and optionally 
+to enable sending your fortune to OctoText. [OctoText](https://plugins.octoprint.org/plugins/OctoText/) must be enabled and configured for
+this feature to work.
+
+## Copyright
+
+Copyright © 2008-2012 Brian M. Clapper. All rights reserved.
```
Comment (jneilliii) [inline _plugins/fortune.md:118]: Should be Copyright © 2008-2019 Brian M. Clapper, based on the referenced website?

**Review by jneilliii — COMMENTED** [review]
@berrystephenw not sure if this was intentional, but seems like there are some bits left over from OctoText in this plugin?

https://github.com/berrystephenw/OctoPrint-Fortune/blob/8774a6bb572ff4e6af0b2606b9482d5a3fb9a554/octoprint_fortune/__init__.py#L135

**Comment by jneilliii** [comment]
> @berrystephenw not sure if this was intentional, but seems like there are some bits left over from OctoText in this plugin?
> 
> https://github.com/berrystephenw/OctoPrint-Fortune/blob/8774a6bb572ff4e6af0b2606b9482d5a3fb9a554/octoprint_fortune/__init__.py#L135

Never mind, I see from the screenshot that it will also send a text if enabled. Still the question remains about the copyright.

File: _plugins/fortune.md (hunk 1 above)
Comment (berrystephenw) [reply _plugins/fortune.md:118]: Ah, I can change the date - the copyright came from his repository where I grabbed the code: https://github.com/bmc/fortune

Comment (berrystephenw) [reply _plugins/fortune.md:118]: change made to the README copyright date.

Comment (berrystephenw) [reply _plugins/fortune.md]: updated fortune.md with 2019 date.

**Review by jneilliii — APPROVED** [review]

**Pull request merged**